pydy/codegen/tests/test_code.py

```diff
@@ -98,7 +98,7 @@ def test_generate_ode_function(self):
         expected_dx = np.array([v, 1.0 / m * (-c * v + m * g - k * x + F)])
 
         rhs_args = {'constants': constants_map,
-                    'specified': {specifieds[0]: F}}
+                    'specified': {next(iter(specifieds)): F}}
```

Comment:
We can leave this, but just to argue some more, :smile_cat:, the previous version of this is much more readable when it was a list.